.rubocop.yml

```diff
@@ -1,6 +1,18 @@
 # Inherit style from another configuration
 inherit_from: .rubocop_todo.yml
 
+ClassLength:
+   Exclude:
+    - 'app/models/ability.rb'
+
+MethodLength: 
+ Exclude:
+    - 'app/models/ability.rb'
+
+AbcSize:
+  Exclude:
+    - 'app/models/ability.rb'
```

Comment:
Same here. Please just disable the cop for the single method, like you did before. Just re-enable the cop at the end of the method.